timify-api/app.js

```diff
@@ -29,8 +30,13 @@ async function taddyGraphqlRequest(query, variables) {
       'X-API-KEY': `${process.env.API_KEY}`,
     }
   });
+  console.log(response.data)
```

Comment:
do we want to keep this log statement? If you want it for testing that's fine but at the very end you should probably clean up log statements that print data (we usually want to leave only error logging)